python/ray/_private/utils.py

```diff
@@ -640,6 +640,23 @@ def get_cgroupv1_used_memory(filename):
         return None
 
 
+def get_cgroupv2_used_memory(stat_file, usage_file):
+    inactive_file_bytes = -1
+    current_usage = -1
+    with open(usage_file, "r") as f:
+        current_usage = int(f.read().strip())
+    with open(stat_file, "r") as f:
+        lines = f.readlines()
+        for line in lines:
+            if "inactive_file" in line:
+                inactive_file_bytes = int(line.split()[1])
```

Comment:
can we also add link to where this logic was from?

Reply:
this is the same logic as in the C++ code, should I also put a link for that?

Reply:
sg!